src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -1427,3 +1269,203 @@ def _InitLib(self):
 
             self._dmLib.pychip_DeviceController_GetLogFilter = [None]
             self._dmLib.pychip_DeviceController_GetLogFilter = c_uint8
+
+            self._dmLib.pychip_OpCreds_AllocateController.argtypes = [c_void_p, POINTER(
+                c_void_p), c_uint64, c_uint64, c_uint16, c_char_p, c_bool, c_bool, POINTER(c_uint32), c_uint32]
+            self._dmLib.pychip_OpCreds_AllocateController.restype = PyChipError
+
+            self._dmLib.pychip_OpCreds_AllocateControllerForCustomCommissioningFlow.argtypes = [
```

Comment:
Why is this a special case? We should be able to make this possible (externally initialized controller credentials) even for non-custom-flow commissioning.